README.md

```diff
@@ -1 +1,15 @@
-# readme take two
+# Stellarpoint
+
+The Stellarpoint Project aims to build a Pose 3D viewer
+which will be beneficial to the Geo’s Pose, 3D, and VPS teams.
+We hope to create a product that can efficiently and accurately
+display Pose data that is visually appealing and user interactive.
+This viewer should allow the Geo Pose team to debug and easily
+visualize pose data, making this project universally beneficial
+and truly impactful. The name 'Stellarpoint' is in reference to
+the massive point clouds we show complementing the pose data.
```

Comment:
I think the name scope is a little narrow, since our project not only visualize 3D point cloud, but also has ability to visualize other kind of sensor data, like imu, pose, etc.

Reply:
Do you recommend us come up with a different name?

Reply:
Or you can think how to elaborate the name to not only limited to point cloud.